content/well-architected-framework/docs/docs/secure-systems/infrastructure/prevent-lateral-movement.mdx

```diff
@@ -137,17 +112,44 @@ identity is a secret. Managing multiple secrets may lead to secret sprawl.
 
 HashiCorp Vault can help practitioners simplify these workloads and centralize
 these secrets.  Vault's secrets engines give teams solutions for various
-use cases trying to manage secrets. 
+use cases, trying to manage secrets. 
```

Comment:
```suggestion
use cases trying to manage secrets. 
```

I don't think the comma is necessary here since it's not a separate phrase, correct?